ember.js

```diff
@@ -1,6 +1,12 @@
 module.exports = {
   env: {
-    browser: true
+    browser: true,
+    embertest: true
   },
-  extends: ['eslint:recommended', 'plugin:ember-suave/recommended', './core.js']
+  globals: {
+    server: true, // mirage
+    withFeature: true, // feature flag
+    withoutFeature: true
```

Comment:
⚠️ This is not native to Mirage.

Reply:
oh~ where did it come from? I don't find it in ember guide, but in [mirage one](http://www.ember-cli-mirage.com/docs/v0.3.x/quickstart/)

Reply:
https://github.com/kategengler/ember-feature-flags

There's only `withFeature`, `withoutFeature` does not exist in the addon.

Reply:
`withoutFeature` is something we have [only in LEAP](https://github.com/search?q=org%3Apeopledoc+withoutFeature++language%3AJavaScript&type=Code), I'm removing it from the PR